css/DayPickerKeyboardShortcuts.scss

```diff
@@ -17,10 +17,11 @@
 .DayPickerKeyboardShortcuts__show {
   width: 22px;
   position: absolute;
+  z-index: 2;
```

Comment:
Why 2 here? Can this be based off of a variable to make the dependency more explicit?